src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/LoggingConstants.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class LoggingConstants
+    {
+        public const string AppConfigLogCategory = "Microsoft.Extensions.Configuration.AzureAppConfiguration";
```

Comment:
I'm thinking we may want `Microsoft.Extensions.Configuration.AzureAppConfiguration.Refresh` as the refresh category.

Reply:
It allows us to add more logs and for the customer to selectively configure refresh logs vs our others.

Reply:
In that case, we might need to create two different ILogger instances during startup - one for startup logs and one for refresh logs. I think it might be better to indicate refresh vs startup logs via the error message so that we don't have to manage two different loggers. What do you think?

Reply:
Loggers are configured by their category name. Error message won't work.

Example: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0#configure-logging 

In that example they configure the hosting lifetime category individually.

Reply:
Updated